===================================================================
RCS file: /cvsroot/libexif/libexif/libexif/exif-data.c,v
retrieving revision 1.67
retrieving revision 1.68
diff -u -r1.67 -r1.68
--- libexif/libexif/libexif/exif-data.c	2005/03/09 22:11:48	1.67
+++ libexif/libexif/libexif/exif-data.c	2005/03/13 19:56:13	1.68
@@ -231,7 +231,7 @@
 		 * the offset must be an even number. If we need to introduce
 		 * a padding byte, we set it to 0.
 		 */
-		if (s & 1) *ds++;
+		if (s & 1) (*ds)++;
 		*d = exif_mem_realloc (data->priv->mem, *d, *ds);
 		if (!*d) {
 			EXIF_LOG_NO_MEMORY (data->priv->log, "ExifData", *ds);
